dataflow/src/domain/mod.rs

```diff
@@ -185,6 +185,7 @@ impl DomainBuilder {
         Domain {
             index: self.index,
             shard,
+            previous_shard: Default::default(),
```

Comment:
This would skip shard 0 initially, but not sure if that matters.

Reply:
No, that's probably fine.